js/simple/modules/iothub_writer.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-let Protocol = require('azure-iot-device-amqp').Amqp;
-let Client = require('azure-iot-device').Client;
+let clientFromConnectionString = require('azure-iot-device-mqtt').clientFromConnectionString;
```

Comment:
Per Damon: AMQP is important for the connection from the gateway to IoT Hub because it can multiplex telemetry from multiple devices onto a single TCP connection. The IoT Hub module in our gateway SDK can be configured to do MQTT instead, but then you don’t get multiplexing—the module will open a new connection for every device. 

I believe for this sample we should switch it back to Amqp.